test/pyjsontestrunner.py

```diff
@@ -6,57 +6,60 @@
 import types
 
 if len(sys.argv) != 2:
-    print "Usage: %s input-json-file", sys.argv[0]
+    print("Usage: %s input-json-file", sys.argv[0])
     sys.exit(3)
-    
+
 input_path = sys.argv[1]
 base_path = os.path.splitext(input_path)[0]
 actual_path = base_path + '.actual'
 rewrite_path = base_path + '.rewrite'
 rewrite_actual_path = base_path + '.actual-rewrite'
 
+
 def valueTreeToString( fout, value, path = '.' ):
-    ty = type(value) 
-    if ty  is types.DictType:
+    ty = type(value)
+    if isinstance(ty, types.DictType):
```

Comment:
I am a bit confused about this since it appears to take fairly archaic but still functional python2 code that doesn't support python 3 (builtin types like `dict` no longer have aliases in the types module) and replace it with code that doesn't work in python2 or python3.

The non archaic way to write this that should work in python 2 or 3 would be to not even use `ty = type(value)` and just write `isinstance(value, dict)`, `isinstance(value, list)`, etc.

Reply:
@dudecc Made the changes as suggested.